primo/opt_model/opt_model.py

```diff
@@ -492,7 +499,8 @@ def min_budget_usage(model):
             )
 
         LOGGER.info(
-            "Added a constraint to ensure that at least 50 percent of the budget is used when the total budget is sufficient to plug all wells."
+            "Added a constraint to ensure that at least 50 percent of the \
```

Comment:
Minor but string concatenation when using parenthesis does not require \. You can simply close the string and open it again on the next line.